### PR TITLE
Rename cliquet.initialize_cliquet in cliquet.initialize

### PR DESCRIPTION
**Pull request by ametaireau**
r? @leplatrem @Natim

### CODE REVIEW

File: cliquet/__init__.py [hunk 1]
```diff
@@ -294,7 +294,15 @@ def includeme(config):
     config.scan("cliquet.views", **kwargs)
 
 
-def initialize_cliquet(config, version=None, project_name=None):
+def initialize_cliquet(*args, **kwargs):
+    message = ('initialize_cliquet is now deprecated. '
```
Comment (Natim) [inline cliquet/__init__.py]: ``cliquet.iniatilize_cliquet`` is now deprecated

Comment (leplatrem) [inline cliquet/__init__.py]: +1 to prefix with module

**Comment by leplatrem** [comment]
r+

**Comment by ametaireau** [comment]
Updated.

**Pull request merged**